tox.ini

```diff
@@ -104,56 +97,24 @@ commands =
 basepython: python3.7
 recreate = True
 deps =
-  pylint~=2.3
-  flake8~=3.7
-  isort~=4.3
-  black>=19.3b0,==19.*
+  -c dev-requirements.txt
+  pylint
+  flake8
+  isort
+  black
```

Comment:
These are listed in dev-requirements, why duplicate them here?

Reply:
Because dev-requirements has more requirements than each tox environment on its own. That's why it's used with `-c` so that only the version requirements are taken, but the targets are not automatically installed. I only moved the versions out of the toxfile with this. If I used `-r`, it would install unnecessary packages for each target (e.g. we don't need pytest or sphinx for the lint toxenv).